File: cognipy/cognipylib/cognipy/models/ConstraintResult.java
package cognipy.models;

import cognipy.*;
import java.util.*;

public class ConstraintResult
{
	/** 
	 Concept for which the constraint was asked
	*/
	private String Concept;
	public final String getConcept()
	{
		return Concept;
	}
	public final void setConcept(String value)
	{
		Concept = value;
	}

	private HashMap<cognipy.cnl.dl.Statement.Modality, ArrayList<String>> Relations;
	public final HashMap<cognipy.cnl.dl.Statement.Modality, ArrayList<String>> getRelations()
	{
		return Relations;
	}
	public final void setRelations(HashMap<cognipy.cnl.dl.Statement.Modality, ArrayList<String>> value)
	{
		Relations = value;
	}

	private HashMap<cognipy.cnl.dl.Statement.Modality, ArrayList<String>> ThirdElement;
	public final HashMap<cognipy.cnl.dl.Statement.Modality, ArrayList<String>> getThirdElement()
	{
		return ThirdElement;
	}
	public final void setThirdElement(HashMap<cognipy.cnl.dl.Statement.Modality, ArrayList<String>> value)
	{
		ThirdElement = value;
	}
}

File: docsrc/jupyter_notebooks/BugReports/cognipy/CogniPyLib/models/ConstraintResult.cs
using System.Collections.Generic;

namespace CogniPy.models
{
    public class ConstraintResult
    {
        /// <summary>
        /// Concept for which the constraint was asked
        /// </summary>
        public string Concept { get; set; }

        public Dictionary<CogniPy.CNL.DL.Statement.Modality, List<string>> Relations { get; set; }

        public Dictionary<CogniPy.CNL.DL.Statement.Modality, List<string>> ThirdElement { get; set; }
    }
}
